### ISSUE TITLE
Benchmark result for an AMD 6800

### ISSUE THREAD

**Issue by ffleader1** [suvojit-0x55aa/mixed-precision-pytorch#6]
I see the readme in this repo does provide the time/epoch for several GPU.
For future reference, here is my result on a Rx 6800, Rocm v5.2.3, Pytorch-Rocm v5.1.1
Time/Epoch: 
- Without mixed percision: 2m1s
- With mixed percision:  1m7s

So apparently using mixed percision on Navi 21 does bring up certain improving in training time too.

**Comment by suvojit-0x55aa** [comment]
Thanks for this @ffleader1, would you happen to have the accuracy scores too. I'll add it to the README with acknowledgement of course.

**Comment by ffleader1** [comment]
> Thanks for this @ffleader1, would you happen to have the accuracy scores too. I'll add it to the README with acknowledgement of course.

Hmm
I did not run for 200 epochs. That would take quite a few hours, and I do need my PC for work. However, I do want to add some benchmark result. I will try to utilize my Pc when I am not actively using it. Will get back to you in a few days.

**Comment by ffleader1** [comment]
Did you use any special param tuning related to the learning rate?
Because after 200 epoch, without mixed percision, I got a 37% accuracy, which is horrendous, with the running command as you specified in the Read.me

**Comment by suvojit-0x55aa** [comment]
Just the script I've put here, Maybe it need some changes due to how training in RX6800 libs are handled ?
